src/misc/setup.cpp

```diff
@@ -1151,6 +1167,16 @@ bool CommandLine::FindInt(char const * const name,int & value,bool remove) {
 	return true;
 }
 
+bool CommandLine::FindIntOption(const std::string &name, int &value, bool remove) {
+	cmd_it it,it_next;
+	if (!IsOption(name, it))
+		return false;
+	it_next=it;++it_next;
```

Comment:
`++it_next` need to be guaranteed valid before incrementing.
In theory `IsOption()` will do this, but suggest asserting before incrementing.

Reply:
Done.